R/vector_control.R

```diff
@@ -155,6 +155,44 @@ throw_away_nets <- function(variables) {
   }
 }
 
+
+
+
+
+#' @title Distribute larval source habitat management
+#' @description simulates larval source management by reducing recruitment to adult female mosquitoes 
+#' from `set_habitat_management` and correlation parameters from
+#' `get_correlation_parameters`
+#'
+#' @param variables list of variables in the model
+#' @param larvi_min is the fractional reduction in adult females joining susceptible population 
+#' @param habitat_management_timesteps the time when this is switched on 
+#' @param parameters the model parameters
+#' @param correlations correlation parameters
```

Comment:
need to add in 'deprec_param' description and what the -1, #rows and #cols mean

same for 'lsm_rate'

Reply:
i have changed these to lsm_rate_alpha and lsm_rate_beta and added description of rows and column